src/api/Drivers/USBMIDI2/Driver/Device.cpp

```diff
@@ -2280,7 +2344,7 @@ Return Value:Amy
         }
 
         // Check if anything leftover to write to USB
-        if (writeBufferIndex)
+        if (writeBufferIndex && !isMoreData) // and there is not more data coming
         {
             // NEED TO FILL REST OF BUFFER WITH NULL
```

Comment:
Is this step a to-do? (filling the rest of the buffer with null)

Reply:
This was a left over comment. Removed